components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/README.md

```diff
@@ -20,7 +20,7 @@ There are some corresponding branches in the repo for each WSO2 API Manager Vers
 |  APIM Version/API | Branch |
 | :------------ |:-------------
 | APIM 2.0.0 - Admin REST API      | apim-2.0.0
-| APIM 2.1.0 - Admin REST API      | master
+| \> APIM 2.1.0 - Admin REST API      | master
```

Comment:
typo?

Reply:
No. 
It is 
> APIM 2.1.0 - Admin REST API      | master

Apim versions greater than 2.1.0